raise ValueError("This package is reserverd for internal usage and cannot be installed from PyPI")
